ghdocs/Contributing/Contributing.md

```diff
@@ -0,0 +1,92 @@
+# [office-ui-fabric-react](../../../README.md) › Contributing
+
+This document describes how to setup your development environment and contribute changes to the **office-ui-fabric-react** project. This document assumes basic working knowledge with Git and related tools. We are providing instructions specific to this project.
+
+## Setup
+
+- Make sure you have a **github** accont. If not, create one.
+- Install **Node.js LTS 8** or greater from [Node.js website](https://nodejs.org/en/).
+
```

Comment:
Install git as well, and also recommend VSCode for text editing.

Reply:
maybe suggest sourcetree as well, ans talk about forking (set up upstream remote)